addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -361,10 +366,10 @@ export function goBackToBlocks(position = "bottom") {
 export function goToTheme(position = "bottom") {
     return [
         {
-            trigger: "#oe_snippets.o_loaded",
+            trigger: ".o-snippets-tabs",
```

Comment:
```suggestion
            trigger: ".o-website-builder_sidebar",
```

Don't you think we need above selector?

Reply:
Both should work fine but .o-website-builder_sidebar checks the entire sliderbar and .o-snippets-tabs checks whether the tab bar (where theme button present) is exist or not. 

IMO - .o-snippets-tabs looks bit specific to me as when clicking on tab buttons the tab should first exist on slider. let me know your views

Reply:
It will work, but I was checking old code, there was ```#snippets_menu``` element, why they have not clicked on that element in old code in that case, so that's why I do not want to change element which is clicked?